server/src/main/java/org/opensearch/action/admin/cluster/snapshots/status/SnapshotStatus.java

```diff
@@ -96,7 +99,12 @@ public class SnapshotStatus implements ToXContentObject, Writeable {
         includeGlobalState = in.readOptionalBoolean();
         final long startTime = in.readLong();
         final long time = in.readLong();
-        updateShardStats(startTime, time);
+        if (in.getVersion().onOrAfter(Version.CURRENT)) {
```

Comment:
While checking this in 'main', please could you help add a TODO comment with the specific version this needs to be replaced with in the follow-up PR? Applicable for all places we have added 'Version.CURRENT'